### DISCUSSION TITLE
Failed to update auth configuration: Failed to fetch

### DISCUSSION THREAD

**Discussion by zozoheir** [supabase/supabase#17290] [Questions] [answered]
Hi everyone, 

Im getting the following error on all changes made in Authentication: google config, additional URLs, Site URL...
The Google project configured for OAuth was deleted on Google dev console, maybe source of the bug.

Issue is now I can't update the config to the new google project...

![Screenshot 2023-09-08 at 6 04 33 PM](https://github.com/supabase/supabase/assets/42655961/9c9a4550-7584-4507-8c4b-b04ad3f85fa1)

**Comment by GaryAustin1** [comment, marked as answer]
I think this is answered on discord.  The solution was a CORs extension in the browser was blocking PATCH requests (which at least one does by default).

**Reply by julien-blanchon** [reply]
Could you please elaborate ?

**Reply by Sri0035** [reply]
Try using different browser, I faced this issue in brave, while its working perfectly in chrome

**Reply by TLiu2014** [reply]
Encountered the same issue in Oct, 2025. Solved by removing my Chrome extension [Allow CORS: Access-Control-Allow-origin](https://chromewebstore.google.com/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf?hl=en).

**Comment by tff2011** [comment]
I’m trying to save Google login authentication on my self-hosted Supabase in a VPS, but I’m getting an error when saving through the UI.

Failed to update auth configuration: API error happened while trying to communicate with the server.

Does anyone know how to solve this?

**Reply by antonsarg** [reply]
I do get the same error in my local dev environment trying to access a storage bucket.

**Reply by dhust** [reply]
I also get this. The weird thing is that there isn't even a link to the auth providers in the sidebar, which makes me wonder. I can't seem to save any settings in any of the providers, because I get the same error. I'm also self-hosting but on my local computer.

**Reply by GaryAustin1** [reply]
> I also get this. The weird thing is that there isn't even a link to the auth providers in the sidebar, which makes me wonder. I can't seem to save any settings in any of the providers, because I get the same error. I'm also self-hosting but on my local computer.

Local hosting and self hosting are set thru configuration files, not the dashboard.

**Reply by antonsarg** [reply]
I have downgraded to version 1.210.0 of the Supabase package until the problem is resolved. The api error message does not occur with this version.

**Reply by 2u841r** [reply]
> 1.210.0

how? is this? 
Supabase Auth (supabase/gotrue:v2.164.0) 
supabase/gotrue:v2.164.0 
or another something?

**Reply by GaryAustin1** [reply]
The dashboard does not handle auth settings in self hosted and local dev.
You have to do that thru settings files.

**Reply by 2u841r** [reply]
> The dashboard does not handle auth settings in self hosted and local dev. You have to do that thru settings files.

any docs/guide/ref? would be very helpful.

**Reply by GaryAustin1** [reply]
I searched with the docs feature at Supabase.com and found:
https://supabase.com/docs/guides/self-hosting/auth/config
https://supabase.com/docs/guides/local-development/cli/config

You did not say if you are self-hosting or using local dev CLI as both are different methods.

**Reply by 2u841r** [reply]
I solved the problem. not deleting the comment, only editing. 
worked version is below.

**Reply by gabryjust04** [reply]
i have the same problem

**Reply by 2u841r** [reply]
> i have the same problem

Hi, I solved this. Now my website is live with supabase auth + google Oauth and many more services. 
You shouldn't wait for Supabase fix this Ui / connectivity issue. 

Just learn basic docker, edit docker-compose.yml 
```
GOTRUE_EXTERNAL_GOOGLE_ENABLED: 'true'
      GOTRUE_EXTERNAL_GOOGLE_CLIENT_ID: http://your-long-url.apps.googleusercontent.com
      GOTRUE_EXTERNAL_GOOGLE_SECRET: your-client-secret
      GOTRUE_EXTERNAL_GOOGLE_REDIRECT_URI: https://supabasekong.yoursite.com/auth/v1/callback 
GOTRUE_SITE_URL: 'https://yoursite.com'
      GOTRUE_URI_ALLOW_LIST: '${ADDITIONAL_REDIRECT_URLS}'
      ADDITIONAL_REDIRECT_URLS: 'https://yoursite.com'
  ADDITIONAL_REDIRECT_URLS: 'http://localhost:3000,https://yoursite.com' 
```
add callback URL on console.cloud.google.com
I noticed- supabasekong should be supabase domain, when I changed to supabase.yoursite.com not worked. 
(since mine is a nextjs project on coolify, so localhost:3000 in  ADDITIONAL_REDIRECT_URLS)
also, I used yoursite.com for main site, and https://supabasekong.yoursite.com for supabase backend control panel. 
I can help you, [discord](https://discord.com/users/580667111253213185)
